features/home/src/main/kotlin/com/teacherworkout/features/home/LandingScreen.kt

```diff
@@ -1,39 +1,111 @@
+    Column(modifier = Modifier.fillMaxSize()) {
+        LazyColumn(
             modifier = Modifier
                 .fillMaxSize()
+                .padding(horizontal = space16dp),
+            verticalArrangement = Arrangement.spacedBy(space16dp),
+            state = listState
         ) {
-            Row(
-                horizontalArrangement = Arrangement.Center,
-                modifier = Modifier
-                    .background(backgroundColor)
-                    .padding(space16dp)
-                    .fillMaxSize()
-            ) {
+            item {
+                Spacer(modifier = Modifier.height(space16dp))
                 Text(
-                    text = stringResource(id = R.string.title_home),
-                    fontSize = MaterialTheme.typography.button.fontSize,
-                    color = MaterialTheme.colors.error,
-                    modifier = Modifier.alignByBaseline()
+                    text = stringResource(id = R.string.discover_lessons_title),
+                    style = TextStyle(
+                        color = MaterialTheme.colors.primary,
+                        fontSize = MaterialTheme.typography.h4.fontSize,
+                        fontWeight = FontWeight.Bold
+                    )
                 )
+                Spacer(modifier = Modifier.height(space8dp))
+                SearchView(
+                    modifier = Modifier.fillMaxWidth(),
+                    value = searchValue,
+                    onValueChange = {
+                        searchValue.value = it
+                        applyFilter(it.text)
+                    },
+                    onClear = {
+                        searchValue.value = TextFieldValue()
+                        applyFilter("")
+                    },
+                    placeholderText = stringResource(id = R.string.search_label)
+                )
+                Spacer(modifier = Modifier.height(space32dp))
+                Text(
+                    text = stringResource(id = R.string.themes_title),
+                    style = TextStyle(
+                        color = MaterialTheme.colors.primary,
+                        fontSize = MaterialTheme.typography.h5.fontSize,
+                        fontWeight = FontWeight.Bold
+                    )
+                )
+            }
+            items(lessonThemes.value.chunked(2)) { oneOrTwoThemes ->
+                Row(horizontalArrangement = Arrangement.SpaceBetween) {
+                    val heightLessonThemeCard = dimensionResource(id = R.dimen.lesson_theme_card_height)
+                    LessonThemeCard(
+                        modifier = Modifier
+                            .height(heightLessonThemeCard)
+                            .weight(1f),
+                        lessonTheme = oneOrTwoThemes[0]
+                    )
+                    Spacer(modifier = Modifier.width(space16dp))
+                    if(oneOrTwoThemes.size == 1) {
```

Comment:
Not sure I understood, What do you mean by oneOrTwoThemes?

Reply:
`lessonThemes.value.chunked(2)` returns a list having one or two items (themes). Couldn't think of a better name. And I thought is is pretty suggestive.